### PR TITLE
Api docs draft

### PR DESCRIPTION
**Pull request by snkinard**
@kevinnayar take a look at the operations for the trip resource and let me know what you think

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+API Docs
+
+Confirms to jsonapi specification: http://jsonapi.org/
```
Comment (kevinnayar) [inline API.md]: conforms

**Comment by kevinnayar** [comment]
Looks good. I was wondering if we should add `users/self`? But it's not crucial atm.

File: API.md (hunk 1 above)
Comment (snkinard) [inline API.md]: dohhhh

**Comment by snkinard** [comment]
@kevinnayar users/self definitely makes sense. We should definitely add that once we figure out auth flow

**Comment by snkinard** [comment]
@kevinnayar have another look at this before I merge. I think we have everything we need (except auth flow) to get started with users and trips. I was very strict about trying to adhere to the "REQUIRED" pieces of jsonapi since I think that's the best way to try out this spec.

Since you last looked there's a new concept of a "relationship", used to define a link between different data resource types. In this case, `users` and `trips` are resources. We describe the user who created a trip using the `author` relationship that exists on the `trips` resource. It's not a data type itself... just a connection between two. here's some sleepy stuff in the jsonapi spec for further reading: http://jsonapi.org/format/#document-resource-object-relationships

let me know if my explanation doesn't make sense

**Comment by snkinard** [comment]
moving docs to the api repo: https://github.com/snkinard/bon-voyage-api

**Pull request closed without merging**